build_all.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# this script replaces the all.yml playbook due to a variable scoping issue
+# that appeared with Ansible 2.0.0.2
+SCRIPT_DIR=$(cd $(dirname $0) && pwd)
```

Comment:
guilty of leaving code in that I was thinking maybe useful later to create an absolute path... I'll purge it np